styles.css

```diff
@@ -7,8 +7,9 @@ body{
   flex-wrap: wrap;
   font-family
 }
-header, footer, aside, .main{
+header, footer, aside, main{
   padding: 1em;
+  margin: 0;
 }
```

Comment:
I don't see any code that applies to styling to the direct children of header, footer, aside and main. You would need to do something like this for each one...

```css
header > * {
  /* style goes here */
}
```
> `>` means direct children and `*` means everything. So the above selector would apply style to everything that is a direct child of `header`.